scopes/ui-foundation/use-box/menu/menu.tsx

```diff
@@ -59,34 +46,18 @@ export function Menu({ packageName, componentId, registryName, componentName, el
           <span>{`Use ${componentName}`}</span>
         </div>
       </div>
-      <Tabs activeTab={activeTab} onClick={setActiveTab} />
-      {(activeTab === 'bit' || !activeTab) && (
-        <Import
-          componentName={componentName}
-          componentId={componentId}
-          packageName={packageName}
-          back={() => setActiveRegistry('import')}
-        />
-      )}
-      {activeTab === 'npm' && (
-        <Install
-          componentName={componentName}
-          registryName={registryName}
-          copyString={`npm i ${packageName}`}
-          packageManager="npm"
-          back={() => setActiveRegistry('install')}
-        />
-      )}
-      {activeTab === 'yarn' && (
-        <Install
-          componentName={componentName}
-          registryName={registryName}
-          copyString={`yarn add ${packageName}`}
-          packageManager="yarn"
-          back={() => setActiveRegistry('install')}
-        />
-      )}
-      {activeTab === 'elements' && elementsUrl && <Elements componentName={componentName} url={elementsUrl} />}
+      <div className={styles.tabs}>
+        {content.map(({ Title, Component: comp }, index) => {
+          if (!Title || !comp) return null;
```

Comment:
you said you will allow undefined for the whole content.. (by using `.filter(Boolean)` or `lodash.compact`)